client-cli/src/main/java/au/org/consumerdatastandards/client/cli/support/ApiUtil.java

```diff
@@ -77,9 +103,25 @@ public static ApiClient createApiClient(ApiClientOptions clientOptions) throws A
         }
 
         String accessToken = clientOptions.getAccessToken();
-        if (StringUtils.isNotBlank(accessToken)) {
+        if (StringUtils.isNotBlank(accessToken) || StringUtils.isNotBlank(clientOptions.getRefreshToken())) {
+            Integer exp = null;
+            Map<String, Object> claims = null;
+            if (StringUtils.isNotBlank(accessToken)) {
+                try {
+                    claims = parseClaims(accessToken);
+                    exp = (Integer) claims.get("exp");
+                } catch (ApiException e) {
+                    LOGGER.info("Invalid access token.");
+                }
+            }
+            if (exp == null || exp.longValue() * 1000 < System.currentTimeMillis() + 10000) {
```

Comment:
Shall we define a constant for 10000 here and give the constant a name like `EXPIRY_BUFFER`?

Reply:
We shall!